src/mixins/input-state/input-state.ts

```diff
@@ -105,6 +105,10 @@ export class InputState extends ModulVue implements InputStateMixin {
     }
 
     public get state(): InputStateValue {
+        if (this.waiting) {
```

Comment:
J'ai bumpé waiting à ce niveau... en ce moment c'était pas possible de mettre un contrôle read-only en waiting.  Donc disons qu'on avait un copy-to-clipboard pendant qu'on fetchait son lien c'était impossible d'afficher le petit spinner dessus

Reply:
Je pense que c'est bien mais on devrait peut être flagger le breaking change UI 

maintenant lorsque le champs est a waiting il ignore tout les autres état (même disabled)

@jipigi

Reply:
Include this section in the release notes
input-state: waiting state will now take precedence over readonly

Reply:
J'ai ajouté le flag